akka-actor-tests/src/test/scala/akka/serialization/SerializeSpec.scala

```diff
@@ -20,7 +20,7 @@ import org.apache.commons.codec.binary.Hex.encodeHex
 
 object SerializationTests {
 
-  val serializeConf = """
+  val serializeConf = s"""
```

Comment:
yes, exactly; this one is quite stupid in that it cannot be switched off, but it has caught a few mistakes for me in the past so I’d like to keep it on